documentation/src/docs/pravega-concepts.md

```diff
@@ -11,362 +11,250 @@ You may obtain a copy of the License at
+- A Stream starts at time **t0** with a configurable number of Segments.  If the
+rate of data written to the Stream is constant, there will be no change in the number of Segments. 
+
+- At time **t1**, the system noted an increase in the ingestion rate and splits **Segment 1** into two parts. This process is referred as **Scale-Up** Event.
+
+- Before **t1**, Events with a Routing Key that hashes to the upper part of the key
+space (i.e., values ranging from **200-399**) would be placed in **Segment 1** and those that hash into the
+lower part of the key space (i.e., values ranging from **0-199**) would be placed in **Segment 0**.
+
+- After **t1**, **Segment 1** is split into **Segment 2** and **Segment 3**. The **Segment 1** is sealed and stops accepting writes.  At this point in time, Events with Routing Key **300** and _above_ are written to **Segment 3** and those between **200** and **299** would be written into **Segment 2**.
+
+- **Segment 0** continues accepting the same range of Events as before **t1**.  
+
+- Another scale-up Event occurs at time **t2**, as **Segment 0**’s range of Routing
+Key is split into **Segment 5** and **Segment 4**. Also at this time, **Segment 0** is sealed
+and allows no further writes.
+
+- Segments covering a contiguous range of the key space can also be merged. At
+time **t3**, **Segment 2**’s range and **Segment 5**’s range are merged to **Segment 6** to
+accommodate a decrease in the load on the Stream.
+
+When a Stream is created, it is configured with a **Scaling Policy** that
+determines, how a Stream handles the varying changes in its load. Pravega has three kinds of Scaling Policy:
+
+1.  **Fixed**:  The number of Stream Segments does not vary with load.
+
+2.  **Data-based**: Pravega splits a Stream Segment into multiple ones (i.e., Scale-up Event) if the number of bytes per second written to that Stream Segment increases beyond a defined threshold. Similarly, Pravega merges two adjacent Stream Segments (i.e., Scale-down Event) if the number of bytes written to them fall below a defined threshold.
+
+3.  **Event-based**:  It is similar to the size-based scaling policy, but it uses number of Events instead of bytes.
 
 ### Events, Stream Segments and AutoScaling
 
 
-We mentioned previously that an Event is written into one of the Stream's
-Segments.  Taking into account AutoScaling, you should think of Stream Segments
-as a bucketing of Events based on Routing Key and time.  At any given time,
-Events published to a Stream within a given value of Routing Key will all appear
-in the same Stream Segment.
+As it was mentioned in the earlier part of the section, that an Event is written into one of the Stream Segments. By considering auto scaling, Stream Segments performs bucketing of Events based on Routing Key and time. It is obvious that, at any given time, Events published to a Stream with a given value of Routing Key will appear in the same Stream Segment.
```

Comment:
As it is a concept in Terminology, please change: `auto scaling` -> `Auto Scaling`

Reply:
done